<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultControlPane</class>
 <widget class="QWidget" name="ResultControlPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>后处理控制面板</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" rowstretch="2,2,3">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>曲线信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>13</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>测试阶段：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_stage">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>数据类型：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_type">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>测试频率：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_freq">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>测试次数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_times">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>显示</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_magn">
           <property name="text">
            <string>幅度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_phase">
           <property name="text">
            <string>相位</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_db_line">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_db">
           <property name="text">
            <string>dB值</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_line">
           <property name="text">
            <string>线性值</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_phase_range">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_360">
           <property name="text">
            <string>0~360</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_0_180">
           <property name="text">
            <string>±180</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>11</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_import">
        <property name="text">
         <string>导入</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_export">
        <property name="text">
         <string>导出</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_clear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_full">
        <property name="text">
         <string>全屏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
